app/services/ItemsCachingService.scala

```diff
@@ -86,31 +86,22 @@ class ItemsCachingService @Inject()(cacheService: CustomsCacheService)(appConfig
       .getEntry[Seq[PackageInformation]](PackageInformation.formId)
       .map(_.zipWithIndex.map {
         case (packageInfo, index) =>
-          createPackaging(packageInfo, index)
+          PackagingBuilder.createPackaging(packageInfo, index)
```

Comment:
Dependency Injection?

Reply:
ItemCachingService will cease to exist soon so no point in changing this.